Develop layout_catalog_grid

sergey-lipikhin opened the pull request:
- [DEMO LINK](https://sergey-lipikhin.github.io/layout_catalog_grid/)

sergey-lipikhin committed:
Readme file initial change

sergey-lipikhin committed:
Solution is finished

sergii-nosachenko commented on src/styles/_catalog.scss:
```diff
@@ -1,22 +1,30 @@
+$card-width: 200px;
```
Create a separate file for variables.

sergii-nosachenko commented on src/styles/_catalog.scss:
```diff
@@ -1,22 +1,30 @@
-    align-items: center;
-    width: 200px;
+    display: grid;
+    place-content: center;
```
🔥

sergii-nosachenko commented on src/styles/_catalog.scss:
```diff
@@ -1,22 +1,30 @@
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: $card-width;
   }
```
You can make a loop to create all media queries. Just create a correctly mapped list to iterate.

sergey-lipikhin committed:
Problems fixed

1. Created file _variables.
2. $card-width renamed to $column-width
3. Media queries created with loop and map list.
